template/nuxt/nuxt.config.js

```diff
@@ -75,14 +75,19 @@ module.exports = {
     '@/plugins/antd-ui'
     <%_ } _%>
   ],
-  <%_ if (eslint) { _%>
   /*
-  ** Nuxt.js devModules
+  ** Nuxt.js dev-modules
```

Comment:
Good catch! Maybe we could remove the whole devModules key in the config object if neither eslint nor tailwind had been selected?

Reply:
Like this?

Reply:
:+1:

Reply:
I think it could be kept, since if we add more devModules in future, the checking will be painful. And keeping an empty `devModules` can be considered as a hint that user may leverage it.

Reply:
I removed the checking for now, we can revert my change if there is any other good reason